pkg/kubelet/sysctl/safe_sysctls.go

```diff
@@ -16,13 +16,17 @@ limitations under the License.
 
 package sysctl
 
-import (
-	"k8s.io/apimachinery/pkg/util/validation/field"
-	api "k8s.io/kubernetes/pkg/apis/core"
-)
-
-// SysctlsStrategy defines the interface for all sysctl strategies.
-type SysctlsStrategy interface {
-	// Validate ensures that the specified values fall within the range of the strategy.
-	Validate(pod *api.Pod) field.ErrorList
+// SafeSysctlAllowlist returns the allowlist of safe sysctls and safe sysctl patterns (ending in *).
```

Comment:
this was hoisted from k8s.io/kubernetes/pkg/security/podsecuritypolicy/sysctl to allow deleting that package